TypeScript.YAML-tmLanguage

```diff
@@ -466,12 +466,12 @@ repository:
 
   case-clause:
     name: case-clause.expr.ts
-    begin: '(?<!\.)\b(case|default)\b'
+    begin: '(?<!\.)\b(case|default(?=:))\b'
```

Comment:
I have tested "export default" which was causing the problem. There are existing tests with switch statement using case and default - and those work as expected.